scripts/build.sh

```diff
@@ -25,9 +25,9 @@ echo "Previous client cleaned."
 
 echo "Build entry point..."
 mkdir -p ./build/server/views
-jade --client --no-debug --hierarchy --out ./build/server ./server
-echo "var jade = require('jade/runtime');module.exports=" | \
-    cat - ./build/server/views/index.js > ./build/server/views/index.js.tmp
+pug --client --no-debug --out ./build/server ./server
```

Comment:
Yep, that's the case, the build script export the `./node_modules/.bin` in front of `$PATH` ([here](https://github.com/cozy-labs/konnectors/pull/555/files/e1238dd339973ebd7fa54650fb9c41aebf0f9f9e#diff-d99b26f864526f6f8f79861d49deb922L3)), so all commands in the script are locals 😃.